mainapp/models.py

```diff
@@ -184,3 +182,43 @@ class DistrictCollection(models.Model):
     collection = models.TextField(
         verbose_name="Details of collected items"
     )
+
+class RescueCamp(models.Model):
+    verbose_name = 'Relief Camp'
+    name = models.CharField(max_length=50,verbose_name="Name")
+    location = models.TextField(verbose_name="Address",blank=True,null=True)
+    district = models.CharField(
+        max_length=15,
+        choices=districts
+    )
+    contacts = models.TextField(verbose_name="Phone Numbers",blank=True,null=True)
+    data_entry_user = models.ForeignKey(User,models.SET_NULL,blank=True,null=True)
+    class Meta:
+        verbose_name = 'Relief Camp'   
+    def __str__(self):
+        return self.name
```

Comment:
Suggestion 

```python
"%s - %s" % (self.name, self.district)
```
To avoid confusing between selecting a district and then selecting a camp in another district
District: `Ernakulam`
Camp: `Themala`
vs
Camp: `Themala - Kollam`